pkg/xds/envoy/routes/reset_tags_configurer.go

```diff
@@ -0,0 +1,19 @@
+package routes
```

Comment:
If you use the code from other PR, please open a PR against that PR. It makes the code review easier because I don't have to reread the code.

So this PR should not be to `master` but to `feat/header-with-tags`, then when `feat/header-with-tags` is merged to master you can rebase and change the target of this PR to `master`